tbl/summary_top_10_invade.tex

```diff
@@ -0,0 +1,11 @@
+\begin{tabular}{lrrl}
+\toprule
+                    Player &  Median $p_1$ &  Memory Depth & Stochastic \\
+\midrule
+        CollectiveStrategy &         0.119 &            \(\infty\) &      False \\
+                  Predator &         0.114 &             9 &      False \\
+   BackStabber: ('D', 'D') &         0.113 &            \(\infty\) &      False \\
```

Comment:
We might consider not using these two strategies since they use knowledge of the tournament length